### ISSUE TITLE
cdkTrapFocus sets initial focus on last item in tab sequence

### ISSUE THREAD

**Issue by diminutivesloop** [angular/material2#10247]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
When tabbing into a region focus trapped via the `cdkTrapFocus` directive the tab order should not change.

#### What is the current behavior?
The last item in the focus-trapped region's tab sequence is focused first. Subsequent tabs follow the expected tab sequence.

#### What are the steps to reproduce?
https://stackblitz.com/edit/angular-material2-issue-prgkm1?file=app%2Fapp.component.html

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
@angular/cdk 5.2.3

**Comment by jelbourn** [comment]
This is working as intended. Focus-trapped regions are not meant to be tabbed into from another region; focus should be manually set to the first element in the region (either with native APIs or the focus-trap's `focusFirstTabbableElement` method).
